compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1774,11 +1774,11 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
       {
       case TR::vadd:
       case TR::vsub:
-         if (dt == TR::Int16 || dt == TR::Int8 || dt == TR::Int32 || dt == TR::Float || dt == TR::Double)
+         if (dt == TR::Int64)
             return true;
-         else
-            return false;
       case TR::vmul:
+         if (dt == TR::Int16 || dt == TR::Int8)
+            return true;
```

Comment:
Can this be changed to list every type that works rather than relying on a fallthrough?